mujoco_py/tests/test_modder.py

```diff
@@ -49,28 +49,28 @@ def test_textures():
     modder.set_rgb('g2', (0, 255, 0))
     modder.set_rgb('g3', (0, 0, 255))
     modder.set_rgb('g4', (255, 0, 255))
-    compare_imgs(sim.render(200, 200, camera_name="topcam"),
+    compare_imgs(sim.render(201, 205, camera_name="topcam"),
```

Comment:
hmm, the issue was with texture sizes, not render buffer sizes, right? why do we need to change these test images?